<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QWidget" name="UserDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 75 16pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nickname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nickname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 75 16pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phonenumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_phonenumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 75 16pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Hannotate SC&quot;;
color: rgb(255, 102, 255);
background-image: url(:/new/prefix1/login.png);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Hannotate SC&quot;;
color: rgb(0, 255, 255);
background-image: url(:/new/prefix1/login.png);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_passwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_passwd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 75 16pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_verify">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 255);
font: 75 16pt &quot;Hannotate SC&quot;;</string>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phoneverifiy">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
